<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密    码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用 户 名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>验 证 码 ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: lightgray;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRePass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtReVer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Register</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Register</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
